MovieViewer/MovieCell.swift

```diff
@@ -134,4 +142,9 @@ class MovieCell: MGSwipeTableCell {
         customSelectionView.backgroundColor = themeColor
         selectedBackgroundView = customSelectionView
     }
+    
+    func didFavorite() {
+        movie?.isFavorited = true
```

Comment:
this should certainly be a `movie!`, right? you cannot favorite the movie if it doesn't exist, something has gone terribly wrong if that happens.